docs/site/submitting_a_pr.md

```diff
@@ -33,7 +33,7 @@ Our commit messages are formatted according to
 [Conventional Commits](https://conventionalcommits.org/).
 
 Please read the
-[Commit Message Format](https://loopback.io/doc/en/contrib/code-contrib-lb4.html#commit-message-guidelines)
+[Commit Message Format](https://loopback.io/doc/en/lb4/code-contrib-lb4.html#commit-message-guidelines)
```

Comment:
IMO, it's even better to reference the md file: [DEVELOPING.md](DEVELOPING.md#commit-message-guidelines)

```
[DEVELOPING.md](DEVELOPING.md#commit-message-guidelines)
```